### DISCUSSION TITLE
Export Route Option

### DISCUSSION THREAD

**Discussion by trymeouteh** [osmandapp/OsmAnd#20157] [Ideas]
I cannot find a export route button in the app, I am easily able to import a route.

Would like an option to export a route and save it as a file anywhere on my device.

**Comment by dmpr0** [comment]
You can export GPX tracks from Track context menu – Share. If you want to share calculated route in Navigation, open Menu – Navigation – Set start and Destination, wait until route will be calculated – tap on Details – You can find Share button in the up right corner.

**Reply by trymeouteh** [reply]
I would never expect to have to export things using the Share button. I think it would enhance the UI experience by adding an export button which will make it clear how to export.

**Reply by albansuser** [reply]
The download/export button (icon) is immediately to the left of the share button referred to above.
